gui/src/controls/image.ts

```diff
@@ -174,6 +174,7 @@ module BABYLON.GUI {
         }
         set cellId(value: number) {
             this._cellId = value;
+            this._markAsDirty();

```

Comment:
Could you check if the _cellId is different before (same pattern used elsewhere in the file like https://github.com/BabylonJS/Babylon.js/pull/3291/files#diff-6d9c4dadf5a981b552ad14470f429d4dR101)

Would be cool to set the same  pattern in cellWidth and cellHeight as well for consistency.